scripts/wrong-filename.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+# SPDX-License-Identifier: MIT
+
```

Comment:
Could use a description of what the script does here :-)

Reply:
Sure, I am working on the remaining entries of the tracker issue now, will do this along with adding documentation for the `scripts` directory.

Reply:
Yeah, I am reworking it to something like this (will check your suggestion too):

```sh
#!/bin/sh
# SPDX-License-Identifier: MIT
#
# This script checks consistency between the filenames and the page
# title.
#
# Usage: ./scripts/wrong-filename.sh

# Output file for recording inconsistencies
OUTPUT_FILE="filename-consistency-check-output.txt"
# Remove existing output file (if any)
rm -f "$OUTPUT_FILE"

set -e

# Iterate through all Markdown files in the 'pages' directories
for path in $(find pages* -name '*.md' -type f); do
  # Extract the expected command name from the filename
  COMMAND_NAME_FILE=$(basename "$path" | head -c-4 | tr '-' ' ' | tr '[:upper:]' '[:lower:]')

  # Extract the command name from the first line of the Markdown file
  COMMAND_NAME_PAGE=$(head -n1 "$path" | tail -c+3 | tr '-' ' ' | tr '[:upper:]' '[:lower:]')

  # Check if there is a mismatch between filename and content command names
  if [ "$COMMAND_NAME_FILE" != "$COMMAND_NAME_PAGE" ]; then
    echo "Inconsistency found in file: $path" >> "$OUTPUT_FILE"
    echo "Expected: $COMMAND_NAME_FILE" >> "$OUTPUT_FILE"
    echo "Actual  : $COMMAND_NAME_PAGE" >> "$OUTPUT_FILE"
    echo "" >> "$OUTPUT_FILE"
  fi
done

echo "Filename consistency check completed. Output written to: $OUTPUT_FILE"
```

I had a second thought so instead of documenting the scripts in separate files I am adding them to the existing files header.

Reply:
Seems like `head -c` doesn't seem to be supported on Mac, will try out with `sed` instead (will DM you the script to test).

Reply:
Done #11810. Feel free to suggest any changes.